.travis.yml

```diff
@@ -4,5 +4,6 @@ node_js:
 sudo: false
 script:
   - npm run lint
+  - cd packages/material-ui-codemod && npm install && cd ../../
```

Comment:
Hey @oliviertassinari , I think if this line is required for `npm run test` to execute without error, that those dependencies need to be installed to a `postinstall` script hook.

Otherwise, we should keep the tests for this separate -- similar to how `icon-builder` has it's own tests.

I'm fine with either one, I just don't like the idea of `npm install && npm run test` failing because the user didn't first navigate to a subdirectory and `npm install`.